packages/devtools_app/lib/src/shared/preferences.dart

```diff
@@ -87,7 +87,7 @@ class InspectorPreferencesController {
   }
 
   /// Change the value for the hover eval mode setting.
-  void toggleHoverEvalMode(bool enableHoverEvalMode) {
+  void setHoverEvalMode(bool enableHoverEvalMode) {
```

Comment:
let's change this back to `toggleHoverEvalMode`. Avoid starting method names with get and set.

Reply:
It is just about get, not set, both for getters and methods: https://dart.dev/guides/language/effective-dart/design#avoid-starting-a-method-name-with-get

I would keep 'set' here. Toggle is confusing, because toggle is expected to read value, switch it and save.